lib/logement.ts

```diff
@@ -57,29 +55,30 @@ function getStatutOccupationLabel(statut) {
   return STATUT_OCCUPATION_LABEL[statut]
 }
 
-function isOwner(logementStatut) {
-  return (
-    logementStatut === "proprietaire" || logementStatut === "primo_accedant"
-  )
+function isOwner(_logementType, _primoAccedant) {
+  return _logementType === "proprietaire" || _primoAccedant === true
 }
 
-function captureCharges(logementStatut) {
+function captureCharges(_logementType, _primoAccedant, _locationType) {
```

Comment:
```suggestion
function captureCharges({_logementType, _locationType}) {
```

C'est plus une remarque qu'une suggestion. Je trouve que c'est plus élégant de tout mettre dans un objet.

Reply:
Je me note ça aussi https://trello.com/c/UvQBIOXk/1032-refactor-lib-logementjscapturecharges